Packages/com.unity.inputsystem/InputSystem/Plugins/Visualizers/InputVisualizer.cs

```diff
@@ -0,0 +1,82 @@
+#if DEVELOPMENT_BUILD || UNITY_EDITOR || PACKAGE_DOCS_GENERATION
```

Comment:
If I have one of these in my scenes, and I make a non-development build, will Unity silently remove it? Or will there be errors?

Reply:
That is a good question. I vaguely remember that we purge all MBs with missing scripts when we do player builds. But I haven't actually gone and made sure. Another reason to put this in the samples, I guess :)

Reply:
So now that we did move them, should we remove this whole `#if`?

Reply:
(Or even if we still think `DEVELOPMENT_BUILD || UNITY_EDITOR` make sense, `PACKAGE_DOCS_GENERATION` does not in the Samples case)

Reply:
Ah yup, those indeed don't make sense anymore. Fixed.